doc/markdown/user/implementation/library/Usage.md

```diff
@@ -26,19 +26,24 @@ ALPAKA_FN_ACC auto doSomethingOnAccelerator(
 Kernel Definition
 -----------------
 
-There is no difference between the kernel entry point function and any other accelerator function in *alpaka* except that it has to return void and the kernel entry point is required to be a function object which has to have the accelerator as first argument.
-This means, the kernel is an object that has implemented the `operator()` member function and can be called like any other function.
-The following code snippet shows a basic example kernel function object.
+A kernel is a special function object which has to conform to the following requirements:
+* it has to fulfill the `std::is_trivially_copyable` trait (has to be copyable via memcpy)
```

Comment:
`memcpy` -> `std::memcpy` (since `std::` is already used in this sentence)?

Reply:
c memcopy as well ;-)

Reply:
Yes, really also C-memcpy.